docs/apps/snakemake.md

```diff
@@ -0,0 +1,58 @@
+---
+tags:
+  - Free
+---
+
+# Snakemake
+
+Snakemake is a python-based scientific workflow management system for creating scalable,
```

Comment:
Here we could maybe also mention that while it is python based it is not only useful for running python scripts but that any scripts can be run with it, but it has special support for R and Python?

Reply:
I wrote a sentence along the lines: "Like in Nextflow, snakemake also supports running scripts (via. script/run/shell directive of snakemake rule) from other languages such as R, bash and Python"